networking/aws_load_balancer_operator/installing-albo-sts-cluster.adoc

```diff
@@ -8,15 +8,41 @@ toc::[]
 
 You can install the AWS Load Balancer Operator on a Security Token Service (STS) cluster.
 
-The AWS Load Balancer Operator relies on `CredentialsRequest` to bootstrap the Operator and for each `AWSLoadBalancerController` instance. The AWS Load Balancer Operator waits until the required secrets are created and available. The Cloud Credential Operator does not provision the secrets automatically in the STS cluster. You must set the credentials secrets manually by using the `ccoctl` binary.
+The AWS Load Balancer Operator relies on a `CredentialsRequest` to bootstrap the Operator and for the `AWSLoadBalancerController` instance. The AWS Load Balancer Operator waits until the required secrets are created and available.
 
 If you do not want to provision credential secret by using the Cloud Credential Operator, you can configure the `AWSLoadBalancerController` instance on the STS cluster by specifying the credential secret in the AWS load Balancer Controller custom resource (CR).
```

Comment:
That's a true statement but we have to back it up with some concrete instructions or remove it for the moment.
As a matter of fact I would really like a follow up PR which would explain how to provide custom credentials secret.

Reply:
Removing this line for now.